src/Tools/ExternalAccess/FSharp/Shared/Options/FSharpServiceFeatureOnOffOptions.cs

```diff
@@ -11,6 +11,6 @@ internal static class FSharpServiceFeatureOnOffOptions
         /// this option doesn't mean we will show all diagnostics that belong to opened files when turned off,
         /// rather it means we will only show diagnostics that are cheap to calculate for small scope such as opened files.
         /// </summary>
-        public static readonly PerLanguageOption<bool?> ClosedFileDiagnostic = Microsoft.CodeAnalysis.Shared.Options.ServiceFeatureOnOffOptions.ClosedFileDiagnostic;
+        public static PerLanguageOption<bool?> ClosedFileDiagnostic = Microsoft.CodeAnalysis.Shared.Options.ServiceFeatureOnOffOptions.ClosedFileDiagnostic;
```

Comment:
❗️ This needs to be turned into a property by changing `=` to `=>`

Reply:
Gotcha